openlibrary/i18n/fr/messages.po

```diff
@@ -245,12 +255,17 @@ msgstr ""
 msgid "Library Explorer"
 msgstr "Explorateur de bibliothèque"
 
-#: lib/header_dropdown.html:46 lib/nav_head.html:107 login.html:10
-#: login.html:72
+#: lib/header_dropdown.html:59 lib/nav_head.html:118 login.html:10
+#: login.html:75
 msgid "Log In"
 msgstr "Se connecter"
 
-#: login.html:15
+#: account/create.html:19 login.html:16
+#, fuzzy
+msgid "OR"
+msgstr "ou"
```

Comment:
@cdrini Do we really want to modify `login-big-or`?
* https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/login.html#L12

Reply:
Yep! We should translate it, it appears on the page here:
![image](https://user-images.githubusercontent.com/6251786/213828984-dcddc6e5-20d5-4834-a133-e98be2e563dc.png)